src/index.tsx

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
 import './index.scss';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
 import configureStore from './configureStore';
-import {Provider} from 'react-redux';
-import {AppInitializer} from './logic/initializer/AppInitializer';
+import { Provider } from 'react-redux';
+import { AppInitializer } from './logic/initializer/AppInitializer';
 
 
 export const store = configureStore();
 AppInitializer.inti();
 
-ReactDOM.render(
-    (<Provider store={store}>
-        <App/>
-    </Provider>),
-    document.getElementById('root') || document.createElement('div') // fix for testing purposes
+const root = createRoot(document.getElementById('root') || document.createElement("div"));
+root.render(
+    <Provider store={store}>
+        <App />
+    </Provider>
 );
 
-serviceWorker.unregister();
+
+// import * as serviceWorker from './serviceWorker';
```

Comment:
Why is that part commented? If we do not need it eny more lets drop it.

Reply:
we haven't needed it yet. the pwa functionality will break everything if the app isn't ready. trust me,😂 I got fired once because of Immature service worker. It ruins the versioning and made everything less robust

Reply:
Co let's remove the comment and forget about PWA. I don't think we need it anyway.

Reply:
now removed.